internal/stackql/acid/binlog/log.go

```diff
@@ -10,7 +8,7 @@ type LogEntry interface {
 	AppendHumanReadable(string)
 	AppendRaw([]byte)
 	Clone() LogEntry
-	Concatenate(LogEntry) LogEntry
```

Comment:
A separate `Clone` method is already provided. So it seems better to have the `Concatenate` method to mutate the `LogEntry` in-place.